quick-start-with-tidb.md

```diff
@@ -6,510 +6,532 @@ aliases: ['/docs/dev/quick-start-with-tidb/','/docs/dev/test-deployment-using-do
-        > + If you want the data to be persisted on storage, run `tiup --tag <your-tag> playground ...`. For details, refer to the [TiUP Reference](/tiup/tiup-reference.md#-t---tag) guide.
+CREATE TABLE hello_tidb (
+     id   INT,
+     name VARCHAR(50)
+  );
 
-4. Start a new session to access TiDB:
+INSERT INTO hello_tidb VALUES
+    (1, 'Hello World');
 
-    + Use the TiUP client to connect to TiDB.
+SELECT * FROM hello_tidb;
+```
 
-        {{< copyable "shell-regular" >}}
+You should see this output:
 
-        ```shell
-        tiup client
-        ```
+```
+SELECT * FROM hello_tidb;
++----+-------------+
+| id | name        |
++----+-------------+
+|  1 | Hello World |
++----+-------------+
+(1 row)
+```
 
-    + Alternatively, you can use the MySQL client to connect to TiDB.
+To exit the client, type `exit` or simply `\q`.
 
-        {{< copyable "shell-regular" >}}
+### Using standard MySQL client
 
-        ```shell
-        mysql --host 127.0.0.1 --port 4000 -u root
-        ```
+You can install the standard MySQL client with:
 
-5. Access the Prometheus dashboard of TiDB at <http://127.0.0.1:9090>.
+<SimpleTab>
+<div label="macOS">
 
-6. Access the [TiDB Dashboard](/dashboard/dashboard-intro.md) at <http://127.0.0.1:2379/dashboard>. The default username is `root`, and the password is empty.
+macOS:
 
-7. Access the Grafana dashboard of TiDB through <http://127.0.0.1:3000>. Both the default username and password are `admin`.
+```shell
+brew install mysql-client
+```
 
-8. (Optional) [Load data to TiFlash](/tiflash/tiflash-overview.md#use-tiflash) for analysis.
+</div>
+<div label="Linux">
 
-9. Clean up the cluster after the test deployment:
+Linux (Debian/Ubuntu):
 
-    1. Stop the above TiDB service by pressing <kbd>Control+C</kbd>.
+```shell
+sudo apt install mysql-client
+```
 
-    2. Run the following command after the service is stopped:
+Linux (RHEL/CentOS):
 
-        {{< copyable "shell-regular" >}}
+```shell
+sudo yum install mysql
+```
+</div>
+</SimpleTab>
 
-        ```shell
-        tiup clean --all
-        ```
+Check if the MySQL client is installed:
 
-> **Note:**
->
-> TiUP Playground listens on `127.0.0.1` by default, and the service is only locally accessible. If you want the service to be externally accessible, specify the listening address using the `--host` parameter to bind the network interface card (NIC) to an externally accessible IP address.
+```shell
+mysql --version
+```
 
-</div>
-<div label="Linux">
+You should see an output indicating the MySQL client version number.
 
-As a distributed system, a basic TiDB test cluster usually consists of 2 TiDB instances, 3 TiKV instances, 3 PD instances, and optional TiFlash instances. With TiUP Playground, you can quickly build the test cluster by following these steps:
+If you already have the MySQL client installed, you can connect to TiDB with:
 
```

Comment:
Do not use multiple consecutive blank lines (line 155~156). Instead, one blank line is enough.
```suggestion
If you already have the MySQL client installed, you can connect to TiDB with:
```